DQMOffline/Configuration/python/autoDQM.py

```diff
@@ -10,6 +10,7 @@
             'outerTracker': ['DQMOuterTracker',
                              'PostDQMOffline',
                              'DQMHarvestOuterTracker'],
+            'lumi': ['DQMOfflineLumi'],
```

Comment:
hi @xmniu - some comments now (more later)

please be sure this new DQM is part of some relval definitions